web-api-db-ctrl/src/postgres.rs

```diff
@@ -67,13 +67,15 @@ const MIGRATION_LOCK_ID: i64 = 0;
 ///
 /// 3. Concurrently for each tenant a migration of their
 ///    schema will be run (if needed).
-#[instrument(skip(pool, legacy_setup), err)] //TODO log enable_legacy_tenant = legacy_setup.is_some
-pub(super) async fn initialize<F>(
+#[instrument(skip_all, err)] //TODO log enable_legacy_tenant = legacy_setup.is_some
+pub(super) async fn initialize<F, F2>(
```

Comment:
:art: for naming consistency i would either name them consecutively or enumerate them all
```suggestion
pub(super) async fn initialize<F, G>(
```
```suggestion
pub(super) async fn initialize<F1, F2>(
```